dolphinscheduler-remote/src/test/java/org/apache/dolphinscheduler/remote/command/log/GetLogBytesRequestCommandTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.remote.command.log;
+
+import org.apache.dolphinscheduler.remote.command.Command;
+
+import org.junit.Test;
+
+import junit.framework.Assert;
+
+public class GetLogBytesRequestCommandTest {
+
+    @Test
+    public void testConvert2Command() {
+        GetLogBytesRequestCommand getLogBytesRequestCommand = new GetLogBytesRequestCommand();
+        getLogBytesRequestCommand.setPath("/opt/test");
+        Command command = getLogBytesRequestCommand.convert2Command();
+        Assert.assertNotNull(command);
```

Comment:
It is better to use an accurate assertion rather than `Assert.assertNotNull`. https://dolphinscheduler.apache.org/en-us/community/development/code-conduct.html
Maybe you can use 
```java
Assert.assertEquals(CommandType.GET_LOG_BYTES_REQUEST, command.getType());
```

Reply:
It seems that you need to declare the test class at pom.xml.

Reply:
> It is better to use an accurate assertion rather than `Assert.assertNotNull`. https://dolphinscheduler.apache.org/en-us/community/development/code-conduct.html
> Maybe you can use
> 
> ```java
> Assert.assertEquals(CommandType.GET_LOG_BYTES_REQUEST, command.getType());
> ```

I've tried using this but it's still not right.